packages/core/src/tools/stackTrace/handlingStack.ts

```diff
@@ -9,14 +9,13 @@ import { computeStackTrace } from './computeStackTrace'
  * - Has to be called at the utmost position of the call stack.
  * - No monitored function should encapsulate it, that is why we need to use callMonitored inside it.
  */
-export function createHandlingStack(): string {
+export function createHandlingStack(framesToSkip = 0): string {
```

Comment:
**💬 suggestion:** ‏It seems like we always skip 2 frames. Maybe we don't need this parameter and we could just use the previous code?